packages/localizations/tsup.config.ts

```diff
@@ -1,68 +1,36 @@
-          'ar-SA': 'src/en-US.retheme.ts',
-          'cs-CZ': 'src/en-US.retheme.ts',
-          'da-DK': 'src/en-US.retheme.ts',
-          'de-DE': 'src/en-US.retheme.ts',
-          'el-GR': 'src/en-US.retheme.ts',
-          'es-ES': 'src/en-US.retheme.ts',
-          'fr-FR': 'src/en-US.retheme.ts',
-          'he-IL': 'src/en-US.retheme.ts',
-          'it-IT': 'src/en-US.retheme.ts',
-          'ja-JP': 'src/en-US.retheme.ts',
-          'ko-KR': 'src/en-US.retheme.ts',
-          'nb-NO': 'src/en-US.retheme.ts',
-          'nl-NL': 'src/en-US.retheme.ts',
-          'pl-PL': 'src/en-US.retheme.ts',
-          'pt-BR': 'src/en-US.retheme.ts',
-          'pt-PT': 'src/en-US.retheme.ts',
-          'ro-RO': 'src/en-US.retheme.ts',
-          'ru-RU': 'src/en-US.retheme.ts',
-          'sk-SK': 'src/en-US.retheme.ts',
-          'sv-SE': 'src/en-US.retheme.ts',
-          'tr-TR': 'src/en-US.retheme.ts',
-          'uk-UA': 'src/en-US.retheme.ts',
-          'vi-VN': 'src/en-US.retheme.ts',
-          'zh-CN': 'src/en-US.retheme.ts',
-          'zh-TW': 'src/en-US.retheme.ts',
-        }
-      : {
-          index: 'src/index.ts',
-          'en-US': 'src/en-US.ts',
-          'ar-SA': 'src/ar-SA.ts',
-          'cs-CZ': 'src/cs-CZ.ts',
-          'da-DK': 'src/da-DK.ts',
-          'de-DE': 'src/de-DE.ts',
-          'el-GR': 'src/el-GR.ts',
-          'es-ES': 'src/es-ES.ts',
-          'fr-FR': 'src/fr-FR.ts',
-          'he-IL': 'src/he-IL.ts',
-          'it-IT': 'src/it-IT.ts',
-          'ja-JP': 'src/ja-JP.ts',
-          'ko-KR': 'src/ko-KR.ts',
-          'nb-NO': 'src/nb-NO.ts',
-          'nl-NL': 'src/nl-NL.ts',
-          'pl-PL': 'src/pl-PL.ts',
-          'pt-BR': 'src/pt-BR.ts',
-          'pt-PT': 'src/pt-PT.ts',
-          'ro-RO': 'src/ro-RO.ts',
-          'ru-RU': 'src/ru-RU.ts',
-          'sk-SK': 'src/sk-SK.ts',
-          'sv-SE': 'src/sv-SE.ts',
-          'tr-TR': 'src/tr-TR.ts',
-          'uk-UA': 'src/uk-UA.ts',
-          'vi-VN': 'src/vi-VN.ts',
-          'zh-CN': 'src/zh-CN.ts',
-          'zh-TW': 'src/zh-TW.ts',
-        },
+    entry: {
+      index: 'src/index.ts',
+      'en-US': 'src/en-US.ts',
+      'ar-SA': 'src/ar-SA.ts',
+      'cs-CZ': 'src/cs-CZ.ts',
+      'da-DK': 'src/da-DK.ts',
+      'de-DE': 'src/de-DE.ts',
+      'el-GR': 'src/el-GR.ts',
+      'es-ES': 'src/es-ES.ts',
+      'fr-FR': 'src/fr-FR.ts',
+      'he-IL': 'src/he-IL.ts',
+      'it-IT': 'src/it-IT.ts',
+      'ja-JP': 'src/ja-JP.ts',
+      'ko-KR': 'src/ko-KR.ts',
+      'nb-NO': 'src/nb-NO.ts',
+      'nl-NL': 'src/nl-NL.ts',
+      'pl-PL': 'src/pl-PL.ts',
+      'pt-BR': 'src/pt-BR.ts',
+      'pt-PT': 'src/pt-PT.ts',
+      'ro-RO': 'src/ro-RO.ts',
+      'ru-RU': 'src/ru-RU.ts',
+      'sk-SK': 'src/sk-SK.ts',
+      'sv-SE': 'src/sv-SE.ts',
+      'tr-TR': 'src/tr-TR.ts',
+      'uk-UA': 'src/uk-UA.ts',
+      'vi-VN': 'src/vi-VN.ts',
+      'zh-CN': 'src/zh-CN.ts',
+      'zh-TW': 'src/zh-TW.ts',
+    },
```

Comment:
Reminder: i will replace it with `entry: ["src/*.ts"]` in a later PRReminder: i will replace it with `entry: ["src/*.ts"]` in a later PR